doc/networking.md

```diff
@@ -196,3 +196,60 @@ for updating Protocol Buffers messages. The same principles of "add but don't
 modify or remove fields" carry over to gRPC service definitions: new services
 can be added, but names should not be changed, nor should services be removed
 unless the intent is to break backwards compatibility.
+
+
+# Redeemer networking spec 
```

Comment:
Let's move this section after the "Ping" section so that it is grouped together with the rest of the networking flow sections. A more descriptive name for this section may be "Remote Ticket Redemption".

Reply:
But it's not necessarily part of the core livepeer networking protocol, should we keep those separate?

Reply:
There are distinct sections for "Broadcaster to Orchestrator" and "Broadcaster to Transcoder" (and there should probably be a "Orchestrator to Transcoder" section added at some point) so adding an additional, but separate section seems reasonable to me. Given the section naming convention, maybe a better section name would be "Orchestrator to Redeemer".

Reply:
fixed in c29f76c